static_code_analysis/.pylintrc

```diff
@@ -0,0 +1,581 @@
+[MASTER]
```

Comment:
Just a small comment, could we move this file into the test directory? I don't like having too many top-level directories